src/blocks/author-carousel/components/AuthorsSelector.js

```
@@ -8,109 +8,133 @@ import { useState } from '@wordpress/element';
+											alt={author.name}
+										/>
+									)}
+									<span>{author.name}</span>
+									<span
+										className="apb-remove-author dashicons dashicons-no-alt"
+										onClick={() => handleRemoveAuthor(id)}
+										title={__(
+											'Remove',
+											'author-profile-blocks'
+										)}
+									></span>
```

ESLint

[error] Visible, non-interactive elements with click handlers must have at least one keyboard listener.

(jsx-a11y/click-events-have-key-events)